base/_buttons.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+// buttons.scss
+// This file contains all styles related to the button element.
+// see http://bitters.bourbon.io
+// dependent on utilities/color.scss
```

Comment:
If this file is dependant on another file for a mixin, extend, or function placing an @import makes sense to me. I like the top level comment heading _dependencies_. We should strive hard to not @import files which generate markup but rather only have mixins and functions.